doc/bridge_packages/flame_rive/rive.md

```diff
@@ -0,0 +1,44 @@
+# flame_rive
+
+`flame_rive` is a bridge library for using [rive](https://rive.app/) animations in your
+Flame game. Rive is a real-time interactive design and animation tool. you can use it to create
+animation
+
+To use it in your game you just need to add `flame_rive` to your pubspec.yaml, as can be seen in the
+[Flame Rive example](https://github.com/flame-engine/flame/tree/main/packages/flame_rive/example)
+and in the pub.dev [installation instructions](https://pub.dev/packages/flame_rive).
+
+
+## How to use
+
+First, add `animation.riv` file in the assets folder. Then load the artboard of the animation to the
+game using the `loadArtboard` method. after, create `StateMachineController` from the loaded
```

Comment:
Same here, remember to capitalize the sentences.
Also the sentence is not grammatically correct.

Reply:
Check the rest of the doc too for non-capitalized sentences, there are several more of them.